packages/scripts/src/monitoring-dvm2.0/voterStakes.ts

```diff
@@ -0,0 +1,59 @@
+// This script verifies the correct functioning of the VotingV2 contract. In particular, it verifies that, once all the
+// voters are processed by the updateTrackers function, the balance of the VotingV2 contract in VotingTokens is equal to
+// the sum of all users' stakes and pendingUnstakes. Otherwise it throws an error.
+// It can be run on a mainnet or goerli fork, with the following command and environment variables:
+// Or a goerli fork with:
+// CUSTOM_NODE_URL=https://<goerli OR mainnet>.infura.io/v3/<YOUR-INFURA-KEY> \
+// yarn hardhat run ./src/monitoring-dvm2.0/voterStakes.ts
+
+const hre = require("hardhat");
+import { VotingTokenEthers, VotingV2Ethers } from "@uma/contracts-node";
+import { getContractInstance } from "../utils/contracts";
+import { forkNetwork, getForkChainId } from "../utils/utils";
+import { getUniqueVoters, updateTrackers } from "./common";
+const { ethers } = hre;
```

Comment:
one small refinement we could make would be to export `ethers`, `VotingTokenEthers, VotingV2Ethers` and prehaps even some of the other utils from `common` and that way you could do a 1 line import here rather than 6 ( and 1 line instead of 4 in the other files)

Reply:
I think this would worsen a bit the readability and make the script more difficult to maintain in the future. For example, if we add any other library or type, it is not very intuitive to have to import it into another file and export it and then import it again. Not strongly opinionated though. Wdyt?